.changeset/fifty-ducks-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@faustwp/core': patch
+---
+
+Export hooks from Faust core
```

Comment:
This should be a minor change since we are introducing something new.

Reply:
@theodesp But there is no breaking changes here. Since we're still 0.x.x, I believe we're still using `patch`